certs/README.md

```diff
@@ -0,0 +1,31 @@
+This directory contains get_certs.sh, for getting a new TLS certificate.
+The script builds the docker image in the root of this repo, then runs
+a command inside it to renew a certificate using certbot.
+
+Requirements
+------------
+The docker container uses aws cli commands, which require credentials
+to be passed into the container via environment variables.
+
+You must set the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment
+variables before running the script, to authenticate as a user with 
+permissions to edit our DNS records.
+
+For example:
+```
+export AWS_ACCESS_KEY_ID=foo
+export AWS_SECRET_ACCESS_KEY=bar
+```
+
+Usage
+-----
+bash get_certs.sh work_dir domain
```

Comment:
Would be better to use: `bash get_certs.sh work_dir domain` here

Reply:
I'm confused -- how is that different from what the README says?

Reply:
Oh I see, you'd like backticks to escape it as code, will do.

Reply:
Sorry I didn't make it clear, yeah adding backticks is what I mean.